lib/IRGen/StructLayout.cpp

```diff
@@ -610,6 +622,17 @@ void irgen::forEachField(IRGenModule &IGM, const NominalTypeDecl *typeDecl,
   }
 }
 
+unsigned irgen::countExportableFields(IRGenModule &IGM,
+                                      const NominalTypeDecl *type) {
+  // Don't count private C++ fields that were imported as private Swift fields
+  int exportableFieldCount = 0;
```

Comment:
nit: 
```suggestion
  unsigned exportableFieldCount = 0;
```

to match return type

Reply:
Oops, thanks!